Data/ByteString/Char8.hs

```diff
@@ -897,6 +897,8 @@ unwords = intercalate (singleton ' ')
 -- | readInt reads an Int from the beginning of the ByteString.  If there is no
 -- integer at the beginning of the string, it returns Nothing, otherwise
 -- it just returns the int read, and the rest of the string.
+--
+-- /Note/: This function will overflow the Int for large integers.
```

Comment:
How large?

Reply:
That naturally depends on what the range of `Int` is on your platform.

If the input is bigger than that, it will overflow.